README.md

```diff
@@ -65,6 +65,20 @@ The containers themselves are running a simple Go web server with five endpoints
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell](https://cloud.google.com/shell/docs/).
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/gke-application-security-demo.git&amp;cloudshell_image=gcr.io/graphite-cloud-shell-images/terraform:latest&amp;cloudshell_tutorial=README.md)
+
+
+All the tools for the demo are installed. When using Cloud Shell execute the following
+command in order to setup gcloud cli. When executing this command please setup your region and zone.
+
+```console
+gcloud init
```

Comment:
Do we need to run gcloud inside cloud shell? I thought it got your CLI set up for the project that you launched it from, but I can't remember.

Reply:
sounds like we do need this one, so no problem here ;)